contracts/interfaces/PolicyI.sol

```diff
@@ -1,7 +1,7 @@
 pragma solidity 0.4.24;
 
 
-contract /* interface */ PermissionPolicyI {
+contract /* interface */ PolicyI {
```

Comment:
Is there a reason for renaming?

Reply:
The policies can also be enforced when appending data, so the permissions label is no longer accurate